.Documentation/articles/innoactive-creator/snapzones.md

```diff
@@ -0,0 +1,96 @@
+
+[![Snap Zone](../images/snap-zones/snapzone-in-action.gif "Snap Zone")](../images/snapzones/snapzone.png)
+
+### Snap Zone Configuration
+
+Snap zones, by default, are not visible to the trainee. However, they are visible for the training designer in the editor window as a volume. 
+
+In order to provide a visual feedback to the trainee when the trainee hovers a snap zone with an interactable object, the snap zone can be configured as follows:
+
+- **Show Interactable Hover Meshes**: If this option is enabled the snap zone will show a preview version of the snappable object.
+- **Interactable Hover Mesh Material**: The material that the preview object will have when a valid snappable object interacts with the snap zone.
+
+It is also possible to configure the snap zone to show a preview version (playmode only) of the desired object to be snap in them as follows:
+
+- **Show Highlight Object**: A prefab that contains only the meshes of the preview object.
+- **Shown Highlight Object Color**: The color that the preview object will have. 
+
+[![Snap Zone Configuration](../images/snap-zones/snapzone.png "Snap Zone Configuration")](../images/snapzones/snapzone.png)
+
+You can create a snap zone either [automatically](#a-automatically-setup-a-simple-snap-zone) (recommended) or [manually](#b-manually-setup-a-simple-snap-zone).
+
+------
+
+## Automatically setup a simple snap zone
+
+### I. Snappable game object
+
+We need a game object that we want to snap into the snap zone. It needs to have at least one collider component. To make it snappable, we need to add the `Snappable Property` component.
+
+[![Add Snappable Property](../images/snap-zones/box-snappable.png "Add Snappable Property")](../images/snapzones/box-snappable.png)
+
+
+### II. Create snap zone automatically
+
+To automatically create a matching snap zone, click the `Create Snap Zone` button of the `Snappable Property` in the `Inspector` window of the corresponding snappable game object.
+
+[![Create Snap Zone](../images/snap-zones/automatically-create.gif "Create Snap Zone")](../images/snapzones/automatically-create.gif)
+
+> The snap zone is generated in the same position, rotation, and with the same volume as the snappable property object.
+
+
+### III. Adjust snap zone (optional)
+
+You can either change the settings directly in the new snap zone object or change the default settings in the `Snap Zone Settings` window. In order to open the default snap zone settings, go to the Unity's toolbar and select: `Innoactive` > `Creator` > `Windows` > `Snap Zone Settings`. All changes are saved automatically, and every snap zone that will be created via the `Create Snap Zone` button afterward will get the new settings by default. The `Apply settings in current scene` button applies these settings to all snap zones in the current scene even if they were not created automatically.
```

Comment:
```suggestion
You can either change the settings directly in the new snap zone object or change the default settings in the `Snap Zone Settings` window. In order to open the default snap zone settings, go to the Unity's toolbar and select: `Innoactive` > `Creator` > `Windows` > `Snap Zone Settings`. All changes are saved automatically, and every snap zone that will be created via the `Create Snap Zone` button afterwards will get the new settings by default. The `Apply settings in current scene` button applies these settings to all snap zones in the current scene even if they were not created automatically.
```

Reply:
What is the difference? 🤔

Reply:
Haha :D interesting

A missing s in "afterwards"